app.js

```diff
@@ -73,15 +73,42 @@ bot.on("text", (ctx) => {
                 //get my batchmates
                 dbo.collection(config.get("mongoCollections.users")).findOne({"from.id" : fromId}, function(error, user) {
                     var reply = "Here are your batchmates : \n";
-                    dbo.collection(config.get("mongoCollections.users")).find({"batch" : user.batch}).forEach(function(batchmate) {
-                        reply += capitalizeFirstLetter(batchmate.from.first_name.toLowerCase()) ;
-                        if( batchmate.from.last_name) {
+                    var count = 1;
+                    dbo.collection(config.get("mongoCollections.users")).find({"batch" : user.batch}).sort({first_name: 1}).forEach(function(batchmate) {
```

Comment:
1. and exclude the user who is making this request.
2. first_name exists inside from object, so it should be 'from.first_name'
3. the code in this file is becoming too much, lets make a different file for all the filters, each filter has its object, containing two functions, one to render output, another to return condition of query to be made, let me start this, then you can edit in that.